src/Admin/PageAdmin.php

```diff
@@ -266,10 +265,10 @@ protected function configureDatagridFilters(DatagridMapper $filter)
                 'field_options' => [
                     'required' => false,
                     'choices' => [
-                        'hybrid' => $this->trans('hybrid'),
-                        'cms' => $this->trans('cms'),
+                        'hybrid' => 'hybrid',
+                        'cms' => 'cms',
                     ],
-                    'choice_translation_domain' => false,
+                    'choice_translation_domain' => 'SonataPageBundle',
```

Comment:
i would have just removed that line and have it get the translation domain from the admin anyway

Reply:
To me, the default value is not the admin translation domain here, but the one from ChoiceType, which is message.

So it's safer this way